src/test/java/tests/TurboMilestoneTest.java

```diff
@@ -34,6 +32,45 @@ public void turboMilestoneTest() {
     }
 
     @Test
+    public void getDueDateComparator_combinationOfDifferentMilestones_correctMilestoneOrdering() {
```

Comment:
I tried to find it in the docs but I can't find it. I wasn't sure about the
Pascal case.
On Mar 16, 2016 11:48 AM, "Damith C. Rajapakse" <notifications@github.com>
wrote:

In src/test/java/tests/TurboMilestoneTest.java
<https://github.com/HubTurbo/HubTurbo/pull/1344#discussion_r56278679>:

> @@ -34,6 +32,45 @@ public void turboMilestoneTest() {
>      }
>
>      @Test
> +    public void getDueDateComparator_combinationOfDifferentMilestones_correctMilestoneOrdering() {

The naming convention is (note that 2nd and 3rd parts are Pascal case so
that the whole thing remains camelCase):

*MethodName_StateUnderTest_ExpectedBehavior*

Following are some of the example:
isAdult_AgeLessThan18_False
withdrawMoney_InvalidAccount_ExceptionThrown
admitStudent_MissingMandatoryFields_FailToAdmit

—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub
<https://github.com/HubTurbo/HubTurbo/pull/1344/files/57817081fe21b55b80051f3222800ea64a6082ea#r56278679>